recipes/plant_tribes_assembly_post_processor/1.0.2/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: plant_tribes_assembly_post_processor
+  version: "1.0.2"
+
+source:
+  fn: v1.0.2.tar.gz
+  url: https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz
+  md5: c0a4217048fe8a18e0c13d67667da638
+
+build:
+  number: 0
+  # Requires genometools which is not supported on osx
+  skip: True # [osx]
+
+requirements:
+  run:
+    - cap3
+    - genometools-genometools >=1,<2
+    - hmmer >=3
+    - mafft >=7,<8
+    - perl >=5.22
+    - perl-estscan2 2.1 pl5.22.0_1
```

Comment:
I agree, its not ideal, but the number of dependencies here results in a pretty complex chain since many involve Perl and some are much more dated than others.  I spent a bunch of time trying to get the combination to work and pinning in this way is what I found to work in the current environment.  I will take look at it when the Perl packages are rebuilt and make sure everything is kosher at that time as well.